CMakeLists.txt

```diff
@@ -42,7 +56,7 @@ ENDIF(HDF5_FOUND)
 # Check for OpenCL
 find_package(OpenCL REQUIRED)
 IF(OpenCL_FOUND)
-  MESSAGE(STATUS "Looking for OpenCL - found at\n   ${OpenCL_INCLUDE_DIRS}\n   ${OpenCL_LIBRARY}")
+  MESSAGE(STATUS "Looking for OpenCL - found at\n   ${OpenCL_INCLUDE_DIRS}\n  ${OpenCL_LIBRARY}")
```

Comment:
Why did you remove the indentation here? Now, it looks a bit weird for me:
```
   /usr/include
  /usr/lib/x86_64-linux-gnu/libOpenCL.so
```

Reply:
It wasn't intentional. VS2017 just likes to changed indentations.